frontend_tests/puppeteer_lib/common.ts

```diff
@@ -482,8 +482,11 @@ class CommonUtils {
         await this.clear_and_type(page, field_selector, str);
         const entry = await page.waitForXPath(
             `//*[@class="typeahead dropdown-menu" and contains(@style, "display: block")]//li[contains(normalize-space(), "${item}")]//a`,
+            {visible: true},
         );
-        await entry!.click();
+        await page.evaluate((entry) => {
+            entry.click();
+        }, entry);
```

Comment:
With the typeahead fix, does the normal `await entry!.click()` work?

Reply:
Nope, it still fails.